packages/manager/modules/sign-up/src/form/activity/controller.js

```diff
@@ -47,6 +48,13 @@ export default class OvhSignUpActivityCtrl {
     }
   }
 
+  resetCorporationData() {
+    this.signUpFormCtrl.model.companyNationalIdentificationNumber = null;
+    this.signUpFormCtrl.model.organisation = null;
+    this.signUpFormCtrl.model.corporationType = null;
+    this.signUpFormCtrl.model.vat = null;
+  }
```

Comment:
**question:** Where is this function used?

Reply:
In the template but it was lost in the refacto... Thx, I'll add it again